snapcraft.yaml

```diff
@@ -813,6 +813,7 @@ parts:
       - --disable-docs
       - --disable-guest-agent
       - --disable-parallels
+      - --disable-pvrdma
```

Comment:
We need to understand impact and what this does.

Reply:
The markdown formatting eat the relevant bit of the commit, I've put them back now.